internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -350,6 +350,12 @@ private ConfigKeys() {
    */
   public static final String CONTROLLER_HELIX_CLOUD_INFO_PROCESSOR_NAME = "controller.helix.cloud.info.processor.name";
 
+  /**
+   * Base URL for customized health checks triggered by Helix. Default is empty string.
+   */
+  public static final String CONTROLLER_HELIX_REST_CUSTOMIZED_HEALTH_URL =
```

Comment:
nit: can we add `check` after health ?

Reply:
I followed the naming convention that Helix has specified: `CUSTOMIZED_HEALTH_URL`. This URL is an umbrella URL, and it doesn't get invoked. Helix will invoke `${CONTROLLER_HELIX_REST_CUSTOMIZED_HEALTH_URL}/aggregatedHealthStatus`. So, the healthcheck part is one API in this umbrella. I'm not sure if we should make this config specific to health check in that case.

What do you think?

Reply:
I see, there could be more endpoints under this.